.github/workflows/inspektor-gadget.yml

```diff
@@ -17,12 +17,12 @@ jobs:
     # level: 0
     runs-on: ubuntu-latest
     steps:
-    - name: Setup go 1.17
+    - name: Setup Go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.17
+        go-version: 1.18
```

Comment:
This breaks "go get" in crd.mk because of the change of semantics in Go 1.18:
https://go.dev/doc/go-get-install-deprecation

It seems to work on the upstream repository but it does not work on a fork:
https://github.com/alban/inspektor-gadget/runs/6900841081?check_suite_focus=true

```
/bin/sh: 1: /home/runner/work/inspektor-gadget/inspektor-gadget/bin/controller-gen: not found
```

I am not sure if it is because something is cached...

I am trying to fix this with something like [this patch](https://github.com/alban/inspektor-gadget/commit/4ea50d19c936466f98e7295e902c4e3ecd1213e9) but there are other errors behind. I am investigating it...

Reply:
The CI tests are fixed with the following commits:
https://github.com/alban/inspektor-gadget/commit/45f31810d4dfcdb120bcc95a741c2b529f247135
https://github.com/alban/inspektor-gadget/commit/b74cdff3eb683499ef79b67c64f865b0f1dc5392